app/src/features/projects/view/ProjectPage.tsx

```diff
@@ -294,16 +294,18 @@ const ProjectPage: React.FC = () => {
                   arrow
                   color="secondary"
                   title={disableDeleteProjectButton ? 'Cannot delete a published project' : ''}>
-                  <Button
-                    variant="outlined"
-                    color="primary"
-                    className={classes.actionButton}
-                    data-testid="delete-project-button"
-                    startIcon={<Icon path={mdiTrashCanOutline} size={1} />}
-                    onClick={showDeleteProjectDialog}
-                    disabled={disableDeleteProjectButton}>
-                    Delete Project
-                  </Button>
+                  <>
```

Comment:
Are these fragment tags needed?

Reply:
yeah, cause the jest tests were giving a console error when the tooltip was given a potentially disabled button as a child

Reply:
Huh, interesting